runtime/common/tests/xcm_tests.rs

```diff
@@ -2134,3 +1816,318 @@ fn test_automatic_versioning_on_runtime_upgrade_with_para_b() {
 		assert!(parachain::para_events().contains(&expected_supported_version_2));
 	});
 }
+
+#[test]
+fn filtered_multilocation_should_not_work() {
```

Comment:
We should cover the two success cases where a user passes the filter.
sending to relay chain - (1, Here)
sending to para chain - (1, X2(para_id, account_id))